lib/tasks/translation.rake

```diff
@@ -85,17 +85,19 @@ namespace :translation do
 
   desc "Add missing translations"
   task(:add_missing, [:locale] => [:environment]) do |t, args|
-    locale = args[:locale] || "en"
+    locale = args[:locale]
     require 'i18n/tasks/cli'
 
-    I18n::Tasks::CLI.start(["add-missing", "-l", locale])
+    i18n_args = ["add-missing", "--nil-value"]
```

Comment:
You've documented why we need the `--nil-value` flag in 4fba60d0426645a084add510f834314e42377792, but the change is introduced here (48a63d9675bfd970f185b1f306c9109d70fab192). Could you move the explanation into this commit?

Reply:
I have moved the commit description.